### PR TITLE
nightlies: build from release branches

### PR DESCRIPTION
**Pull request by scpeters**
Since we have created the release branches, we should build the nighly debs from these. Some nightly builds have started failing because the `main` branches have had some major version bumps merged.

We'll need to update the homebrew formulae as well.

### CODE REVIEW

**Review by j-rivero — APPROVED** [review]

**Comment by scpeters** [comment]
https://github.com/gazebo-tooling/release-tools/pull/785/commits/c4b5b7cb15ecab9446cda0b6fea54202c5fcd9c2: I've reordered the builds so that gz-cmake is before gz-tools

File: jenkins-scripts/dsl/ignition_collection.dsl [hunk 1]
```diff
@@ -18,22 +18,22 @@ ignition_collections = [
...
+          'common'    : [ debbuild: 'ign-common5'     , branch: 'gz-common5' ],
+          'msgs'      : [ debbuild: 'ign-msgs9'       , branch: 'gz-msgs9' ],
+          'rendering' : [ debbuild: 'ign-rendering7'  , branch: 'gz-rendering7' ],
+          'sdformat'  : [ debbuild: 'sdformat13'      , branch: 'sdf13' ],
+          'fuel-tools': [ debbuild: 'ign-fuel-tools8' , branch: 'gz-fuel-tools8' ],
+          'transport' : [ debbuild: 'ign-transport12' , branch: 'gz-transport12' ],
+          'gui'       : [ debbuild: 'ign-gui7'        , branch: 'gz-gui7' ],
+          'sensors'   : [ debbuild: 'ign-sensors7'    , branch: 'gz-sensors7' ],
+          'physics'   : [ debbuild: 'ign-physics6'    , branch: 'gz-physics6' ],
+          'gazebo'    : [ debbuild: 'ign-gazebo7'     , branch: 'gz-sim7' ],
```
Comment (chapulina) [inline jenkins-scripts/dsl/ignition_collection.dsl:35]: This branch doesn't exist yet, we're waiting for the forward-port.

**Review by chapulina — CHANGES_REQUESTED** [review]

File: jenkins-scripts/dsl/ignition_collection.dsl (hunk 1 above)
Comment (scpeters) [reply jenkins-scripts/dsl/ignition_collection.dsl]: https://github.com/gazebo-tooling/release-tools/pull/785/commits/69934c4b4e7a5b58cf2b5443bbf628a628b8ff1b

**Review by chapulina — APPROVED** [review]
Thanks for sorting this out 👍🏽

**Comment by chapulina** [comment]
> We'll need to update the homebrew formulae as well.

* https://github.com/osrf/homebrew-simulation/pull/1985

**Pull request merged**